README.rst

```diff
@@ -38,37 +39,38 @@ Get the authorization url
 
     from boxsdk.auth.oauth2 import OAuth2
 
-    oauth = OAuth2(
-        client_id='YOUR_CLIENT_ID',
-        client_secret='YOUR_CLIENT_SECRET',
-        store_tokens=your_store_tokens_callback_method,
-    )
```

Comment:
The SDK coding style actually prefers the current way because changes are much easier to track, and it still conforms to PEP8. So let's leave this example as is.